Java/core/src/main/java/com/amazon/randomcutforest/state/RandomCutForestState.java

```diff
@@ -30,6 +30,8 @@
 @Data
 public class RandomCutForestState {
 
+    private String version = "compact";
```

Comment:
I think we should use number versions, and we should store them either as String constants or as an enum so that we use a consistent scheme in all the different files. E.g.
```
package com.amazon.randomcutforest.state;

public class Version {
  public static final String V2_0 = "2.0";
}
```

Reply:
Done